karmada-CLI-installtion-and-deploy-example/finish.md

```diff
@@ -0,0 +1 @@
+**The end**
```

Comment:
It'd be great to have a summarize of this playground and make rooms for guiding people to other scenarios.

Reply:
I will add the summary section.
Do you mean to switch to other scenes in Karmada? Other scenarios have not yet been implemented, and the website provides links to other scenarios in Killercoda.

Reply:
For now, since we don't have any other scenarios yet, so a summary would be fine.
But in the future, we might need to add links to other scenarios for people to experience other features.